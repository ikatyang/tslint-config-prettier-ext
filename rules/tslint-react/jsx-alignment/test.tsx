const bad = <Foo bar
    car
/>;
