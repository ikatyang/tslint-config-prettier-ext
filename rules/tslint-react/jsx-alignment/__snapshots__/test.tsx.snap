// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const bad = <Foo bar
    car
/>;

======
const bad = <Foo bar car />;

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (jsx-alignment) /rules/tslint-react/jsx-alignment/test.tsx[1, 18]: JSX attributes must be on a line below the opening tag
ERROR: (jsx-alignment) /rules/tslint-react/jsx-alignment/test.tsx[2, 5]: JSX attributes must be on their own line and vertically aligned
"
`;
